website/content/docs/v0.8/Single Board Computers/libretech_all_h3_cc_h5.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Libre Computer Board ALL-H3-CC"
+---
+
+## Generating the Image
+
+```bash
+docker run \
+  --rm \
+  -v /dev:/dev \
+  --privileged \
+  ghcr.io/talos-systems/installer:latest image --platform metal --board libretech_all_h3_cc_h5 --tar-to-stdout | tar xz
```

Comment:
I wonder if there is a better way ... having to uncompress and untar are extra steps we could eliminate?

Reply:
I think I would like the idea of having a shell script somewhere in the repo to just:

```bash
curl -sLf https://raw.githubusercontent.com/.../gen-image.sh | bash -s -- -o /dev/mmcblk0
```
And the script can do both steps inside.
It doesn't really solve tar untar problem, but at least can simplify the flow.

Reply:
Good idea. I feel like it could be better to host on Talos.dev (nice short url). Something we can probably figure out later? For now I was more concerned about not having to tar/untar.

Reply:
Regarding tar/untar: nothing comes to mind except mounting yet another volume and dumping that data without compression there.

Reply:
I think what we should do is think this through a bit and follow up. I am over it now :smile: .